forage-android/src/main/java/com/joinforage/forage/android/network/OkHttpClientBuilder.kt

```diff
@@ -18,7 +18,17 @@ internal object OkHttpClientBuilder {
                         .addHeader(
                             ForageConstants.Headers.AUTHORIZATION,
                             "${ForageConstants.Headers.BEARER} $sessionToken"
-                        ).run {
+                        )
+                        .run {
+                            chain.request().headers[ForageConstants.Headers.API_VERSION]?.let {
+                                this
+                            }
+                                ?: addHeader(
+                                    ForageConstants.Headers.API_VERSION,
+                                    "default"
+                                )
+                        }
+                        .run {
```

Comment:
can you add a comment to explain what's going on in this block pls

Reply:
Done: [19bcdaf](https://github.com/teamforage/forage-android-sdk/pull/238/commits/19bcdafe2bebf8b9e8b7ab1bff69dbf4e58cc41a) also tested all flows locally.